src/docker/Dockerfile

```diff
@@ -32,6 +32,7 @@ LABEL Description="This is the image for aether.jcabi.com" Vendor="jcabi.com" Ve
 WORKDIR /tmp
 
 ENV DEBIAN_FRONTEND=noninteractive
+ENV OPENJDK_VERSION 1.7.0_75
```

Comment:
@mkordas Version of ``openjdk-7-jdk`` is ``7u111-2.6.7-0ubuntu0.14.04.3``, so I locked that one